vars/apmCli.groovy

```diff
@@ -44,20 +41,9 @@ def call(Map args = [:]) {
     spanName = spanCommand
   }
 
-  def setupPy = libraryResource("scripts/apm-cli/requirements.txt")
-  def apmCliPy = libraryResource("scripts/apm-cli/apm-cli.py")
   dir('apm-cli'){
-    log(level: 'DEBUG', text: "apmCli: Installing APM CLI")
-    writeFile file: "requirements.txt", text: setupPy
-    writeFile file: "apm-cli.py", text: apmCliPy
-
-    if(!url && !token){
-      def apmJson = getVaultSecret(secret: "${apmCliConfig}")?.data.value
-      def apm = readJSON(text: apmJson)
-      url = apm.url
-      token = apm.token
-    }
-
+    def apmCliRelease = "https://github.com/elastic/apm-pipeline-library/releases/download/apm-cli-0.0.1/apm-cli-0.0.1.tar.gz"
```

Comment:
it directly installs the release from GitHub

Reply:
I'd say to add the `retryWithSleep` step